src/App.jsx

```diff
@@ -5,18 +5,21 @@ import Nav from "./components/Nav";
 import User from "./components/user/User";
 import Article from "./pages/Article";
 import Topic from "./pages/Topic";
+import { useState } from "react";
 
 function App() {
+	const [user, setUser] = useState({});
```

Comment:
Would be better to handle this with useContext, so you dont have to keep passing it down from the app.